ts/packages/anchor/src/idl.ts

```diff
@@ -67,7 +67,38 @@ export type IdlPda = {
   programId?: IdlSeed;
 };
 
-export type IdlSeed = any; // TODO
+// Based on the rust type defined in lang/sync/src/idl/types.rs and the definitions in lang/syn/src/idl/pda.rs
+export type IdlSeed =
+  | IdlSeedConst
+  | IdlSeedArg
+  | IdlSeedAccount;
+
+export type IdlSeedConst = {
+  kind: "const";
+  type: IdlType;
+  value: SerdeJsonValue;
+};
+
+export type SerdeJsonValue =
+  | null
+  | boolean
+  | number
+  | string
+  | SerdeJsonValue[]
+  | { [key: string]: SerdeJsonValue };
```

Comment:
Thanks for adding this but I think we can just use `any` here and remove this type.

Reply:
sure!